cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/type/impl/CassandraCalendarDateType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.cassandra.type.impl;
+
+import org.hibernate.MappingException;
+import org.hibernate.engine.spi.Mapping;
+import org.hibernate.ogm.type.impl.AbstractGenericBasicType;
+import org.hibernate.type.descriptor.java.CalendarDateTypeDescriptor;
+
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Cassandra doesn't do Calendar, but it does do Date, so convert them.
```

Comment:
yup, at the cost of giving up any server side query ability, since CQL3 lacks the date manipulation functions that SQL has. May still be worth it though.  I'm inclined to go with what we've got for now and revisit it once the new types are available in CQL.